vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -78,9 +79,6 @@ To add a `Figure` to your page:
         Vizro().build(dashboard).run()
         ```
 
-        1. This creates a [`layout`](layouts.md) with five rows and four columns. The KPI card is positioned in the first cell, while the remaining cells are empty.
-        2. For more information, refer to the API reference for the  [`kpi_card`](../API-reference/figure-callables.md#vizro.figures.kpi_card).
-
```

Comment:
See comment on slack - sometimes we remove the annotations as we do here and add them as code comments, while in other examples we leave it as is. I am not sure what's the best way to solve for this. Here I agree with you that code comments would suffice, but I would add them directly above e.g. the comment on the layout should be either directly above or next to the code line.